Lib9c/Model/Order/FungibleOrder.cs

```diff
@@ -130,9 +129,54 @@ public override OrderDigest Digest(AvatarState avatarState, CostumeStatSheet cos
                 $"Aborted because the tradable item({TradableId}) was failed to load from avatar's inventory.");
         }
 
-        public override void ValidateCancelOrder(AvatarState avatarState, Guid tradableId)
+        public override OrderReceipt Transfer(AvatarState seller, AvatarState buyer, long blockIndex)
+        {
+            if (seller.inventory.TryGetLockedItem(new OrderLock(OrderId), out var inventoryItem))
+            {
+                var tradableItem = (TradableMaterial) inventoryItem.item;
+                seller.inventory.RemoveItem(inventoryItem);
+
+                var copy = (TradableMaterial) tradableItem.Clone();
+                copy.RequiredBlockIndex = blockIndex;
+                buyer.UpdateFromAddItem(copy, ItemCount, false);
+                return new OrderReceipt(OrderId, buyer.agentAddress, buyer.address, blockIndex);
+            }
+
+            throw new ItemDoesNotExistException(
+                $"Aborted because the tradable item({TradableId}) was failed to load from seller's inventory.");
+        }
+
+        public override int ValidateTransfer(AvatarState avatarState, Guid tradableId,
+            FungibleAssetValue price, long blockIndex)
+        {
+            var errorCode =  base.ValidateTransfer2(avatarState, tradableId, price, blockIndex);
```

Comment:
is it okay we invoke `base.ValidateTransfer2()` instead of `base.ValidateTransfer()` in `.ValidateTransfer()`?